src/plots/cartesian/graph_interact.js

```diff
@@ -309,6 +309,13 @@ fx.unhover = function (gd, evt, subplot) {
 // The actual implementation is here:
 
 function hover(gd, evt, subplot){
+    if(gd._fullLayout._hasPie){
```

Comment:
@mdtusz you're right.

The concept of (sub)plots doesn't extend to pies. Pies are considered traces that have thier own domain and no corresponding axes.

What does `evt` contain here?